lib/new_relic/agent/instrumentation/grape.rb

```diff
@@ -9,7 +9,7 @@
 DependencyDetection.defer do
   # Why not just :grape? newrelic-grape used that name already, and while we're
   # not shipping yet, overloading the name interferes with the plugin.
-  named :grape_instrumentation
+  @name = :grape_instrumentation
```

Comment:
All of the instrumentation files that had `named` changed to `@name=` is due to a test we have that automatically checks to make sure all files using `named` have a config that matches the provided name. Since the the config for grape is `instrumentation.grape`, the test was failing because it was checking for `instrumentation.grape_instrumentation`.